scripts/finetune.py

```diff
@@ -161,13 +161,30 @@ def train(
 
     validate_config(cfg)
 
+    # load the tokenizer first
+    logging.info("loading tokenizer...")
+    tokenizer = load_tokenizer(
+        cfg.base_model_config,
+        cfg.tokenizer_type,
+        cfg
+    )
+
+    if "inference" not in kwargs and "shard" not in kwargs:  # don't need to load dataset for these
```

Comment:
You need to also add `merge_lora` here.

Perhaps can be refactored to use the below as the list expand.. Any element in first list is not in second.

```python
def check_lists(list1, list2):
    return not any(element in list2 for element in list1)
```

Reply:
so like `check_lists(["inference", "shard", "merge_lora"], kwargs.keys()]`?

Reply:
Yes or just `kwargs`! And also with a better function name :')